techjb/Vector-Tiles-Google-Maps
# BMAPS-1694 Added @mapbox/vector-tile dependency

## Issue Link
[BMAPS-1694](https://northpoint-development.atlassian.net/browse/BMAPS-1694)

## Description
- This does not change any functionality, but rather, removes three files: `VectorTile.js`, `VectorTileFeature.js`, and `VectorTileLayer.js`.
- Inside `MVTSource.js`, imported `VectorTile` from the `@mapbox/vector-tile` package, which has all the functionality the three removed files contained

**NOTE:** No tests were added, this just changes where the `new VectorTile()` code comes from in `MVTSource.js`  

## QA (By code reviewer)
- Pull down this branch
- Inside the `basic.html` example, change the api key on line 47 to be the key we use in `index.html` of Beyond Maps
- Open the `basic.html` example using Live Server and ensure everything loads properly
